lib/service-util.ts

```diff
@@ -66,6 +66,8 @@ export class ServiceProxy implements Server.IServiceProxy {
 			} catch(err) {
 				if (err.response && err.response.statusCode === 401) {
 					this.$userDataStore.clearLoginData().wait();
+				} else if (err.response && err.response.statusCode === 402) {
```

Comment:
Maybe we can use switch in case it is expected to handle other codes in the future:
```Typescript
if (err.respone) {
    switch(err.response.statusCode) 
 ....
}